solana/client/types/transaction.go

```diff
@@ -23,7 +23,7 @@ type UiTransaction struct {
 }
 
 type TransactionTokenBalance struct {
-	AccountIndex  uint8         `json:"accountIndex"`
+	AccountIndex  uint          `json:"accountIndex"`
```

Comment:
Cause uint8 only serves the number under 256. To work properly in bank module, it is changed into uint, instead.